Add `Locus 2025 Update #1` news post (#13140)
ppy/osu-wiki: news/2025/2025-04-14-locus-2025-update-1.md

Registrations will continue to remain open until the submission deadline of 1 June 2025. If you are interested in participating or have not yet registered a team, we highly recommend you read the [wiki article](/wiki/Contests/Locus/2025) for more details.

While participants have been busy "cliquing" their circles, the Locus team has tirelessly been working behind the scenes on a series of relevant side projects.

### Trial voting

On 16 March, we unveiled the *trial voting* — a demonstration of the community voting system to be used in Locus. The main goal of the trial is to increase transparency on how our voting system works, and to show that it was capable of generating agreeable results.

For this trial, we asked the question:

> What was the best osu! original tiebreaker from the osu! World Cups?

Here are current standings for the top 5 entries:

![Locus trial voting leaderboard](/wiki/shared/news/2025-04-14-locus-2025-update-1/trial-leaderboard.jpg)

[Click here to check out the full leaderboard!](https://locus.yui.tv/leaderboard)

> The voting will remain open until the next news post update, so head over to the [trial voting website](https://locus.yui.tv/) to cast your vote now if you haven't already! For feedback, please use the `#osu-locus` on the [osu! Discord server](https://discord.com/invite/ppy). Your input is very much appreciated!

Reviewer: ```suggestion
The voting will remain open until the next news post update, so head over to the [trial voting website](https://locus.yui.tv/) to cast your vote now if you haven't already! For feedback, please use the `#osu-locus` channel in the [osu! Discord server](https://discord.com/invite/ppy). Your input is very much appreciated!
```

### Demo entry: cygnus - Metamorphism

Before we even announced the contest, members of the Locus team decided it would be a good idea to create a "demo entry", even though we weren't exactly sure what purpose it would have fulfilled at the time.

Ultimately, we felt it would be useful as a general reference. We made a short video documenting the creative process behind our entry, and we would like to encourage other teams to do something similar!

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/2025-04-13-locus-demo.mp4" type="video/mp4" preload="none">
    </video>
</div>

You can download this demo entry [here](https://osu.ppy.sh/beatmapsets/2354848#osu/5070094). Note that it will not be competing in the actual Locus contest.

Many thanks to [Matrix](https://osu.ppy.sh/users/5052899) for coordinating this.

---

That's it from us for now! Please stay tuned for further updates — our next news post will aim to cover teaser #2 and provide instructions for submitting your Locus entry, among other things.

> **Announcements for the contest will be coordinated through the `#locus` channel on the [osu! Discord server](https://discord.com/invite/ppy).**

Reviewer: ```suggestion
**Announcements for the contest will be coordinated through the `#osu-locus` channel in the [osu! Discord server](https://discord.com/invite/ppy).**
```

> —mango, Matrix, and a bunch of other people.

Reviewer: ```suggestion
—mango, Matrix, and a bunch of other people
```
additionally, thoughts on just combining this as "Locus Team"?
